src/main/kotlin/model/CarsManager.kt

```diff
@@ -0,0 +1,32 @@
+package model
+
+import util.NumberGenerator
+import util.Validation
+
+class CarsManager(carsName: List<String>, private val randomNumber: NumberGenerator) {
+
+    var cars: List<Car>
+        private set
```

Comment:
cars를 val로 변경하면 어떻게 될까요?
```suggestion
    val cars: List<Car>
        private set
```